Makefile

```diff
@@ -78,6 +78,11 @@ ifeq ($(USE_PROFILER), 1)
 	CFLAGS += -DMXNET_USE_PROFILER=1
 endif
 
+# CFLAGS for segfault logger
+ifeq ($(USE_SEGFAULT_LOGGER), 1)
+	CFLAGS += -DMXNET_USE_SEGFAULT_LOGGER=1
```

Comment:
let's call it USE_SIGNAL_HANDLER. We may handle more signals in the future

Reply:
do we want to create coupling among all signal handlers?